### ISSUE TITLE
Ability to resize tab title text?

### ISSUE THREAD

**Issue by neverxxever** [Croydon/vertical-tabs-reloaded#195]
Add-on version: 0.11.0
Firefox version: 61.0.1
OS + OS-Version: Ubuntu 16.04 LTS
Other tab + user interface related add-ons: N/A

Hi there,

I've looked everywhere (as far as I can tell) and there doesn't seem to be a way to change the font size for tab titles in VTR. Am I missing something?

I use VTR on my desktop and I'm in the process of migrating my htpc over as well, but the tab title text is tiny by default, to the point where I can barely read it.

Fonts and basic colour options for background etc. would be a nice addition too, but functionally the only thing I really need is font size.

Great add-on overall though, was the only add-on that allowed me to leave Chrome and SideTabs.

Thanks

**Comment by Croydon** [comment]
This is definitely on the to do list, but is part of the larger efforts to customize the appearance of VTR. See #40 and #197

**Comment by neverxxever** [comment]
Ok cool awesome.

For anyone else, in the mean time I found a workaround:
By going to Options and going to 'Advanced' under the Font settings, one can set the minimum size of text in Firefox which will effectively resize the VTR tab title text.

**Comment by HeRo002** [comment]
> Ok cool awesome.
> 
> For anyone else, in the mean time I found a workaround:
> By going to Options and going to 'Advanced' under the Font settings, one can set the minimum size of text in Firefox which will effectively resize the VTR tab title text.

The workaround is OK, but it destroys the layout of many websites, so that you can't really read text that has been allocated a limited amount of space.

**Comment by Croydon** [comment]
Since this is often requested I will make the font size bigger for all themes in 0.13.0. An user setting will still take time however.

**Comment by Crystal-RainSlide** [comment]
Why not use reletive length unit by default???

https://developer.mozilla.org/en-US/docs/Web/CSS/length#Relative_length_units

**Comment by amurzeau** [comment]
The availability of this option is interesting as users with a small screen resolution won't like much the new release I guess which use a bigger font and makes the text not vertically centered on each tab.
Or something that would take into account the dpi settings, but I don't know what's the firefox behavior regarding this.

**Comment by jiri-h** [comment]
> 
> 
> The availability of this option is interesting as users with a small screen resolution won't like much the new release I guess which use a bigger font and makes the text not vertically centered on each tab.
> Or something that would take into account the dpi settings, but I don't know what's the firefox behavior regarding this.

Yes, that is why I got here, googling the "too large font" issue. I have noticed the new update introduced a tab labels that are so big that it impacts my ability to read them and navigate quickly. I have found out that the Darwin theme fortunately has smaller font so I have switched to this one, but having the option to change font size would be much appreciated. Or maybe a workaround - create let's say 5 versions of each theme with different font sizes (smallest-small-normal-large-largest). This quick'n'dirty solution could even prove to be a permanent solution to the font size problem...

**Comment by duncanawoods** [comment]
@Croydon thank you for your hard work for the extension!

Please consider the new larger font size a critical issue. It is so hard and unpleasant to read that I can't bear to use it anymore.

Can you revert the large size asap rather than treat this more ambitious issue as the solution?
